project/BuildDependencies/bin/.wget-hsts

```diff
@@ -0,0 +1,7 @@
+# HSTS 1.0 Known Hosts database for GNU Wget.
```

Comment:
What's the reason for this file?

Reply:
Added automatically by Git when I pulled. No idea why